src/evaluator.rs

```diff
@@ -146,11 +159,33 @@ mod tests {
         return_value: 7,
     }
 
-    evaluator_test_case! {
+    positive_evaluator_test_case! {
         name: assorted_let_with_return_identifier,
         input: r#"
             let a = 42;
             a;"#,
         return_value: 0, // not 42
     }
+
+    macro_rules! negative_evaluator_test_case {
+        (name: $test_name:ident, input: $input:expr, return_value: $return_value:expr,) => {
+            #[test]
+            fn $test_name() {
```

Comment:
can we get 'error' into the name somehow?

Reply:
Done